resourcemonitor/resourcemonitor.go

```diff
@@ -72,6 +72,7 @@ type MonitoringSender interface {
 type TrafficMonitoring interface {
 	GetSystemTraffic() (inputTraffic, outputTraffic uint64, err error)
 	GetInstanceTraffic(instanceID string) (inputTraffic, outputTraffic uint64, err error)
+	UpdateIptablesFilterCache() error
```

Comment:
Add more info in commit message why this API is needed and what it actually does.

Reply:
Add descriptions to commit message

Reply:
I think we should not expose implementation details to the interface. Resource monitoring should not care about updating iptables filter cache. It should not know that monitoring is done using iptables and so on. Probably we can rename this method to something like `PrepareTrafficMonitoring` but a better solution is to do everything in the network manager.